src/lib/ks/interactive

```diff
@@ -134,6 +134,9 @@ if [ -f /etc/nethserver-release ]; then
   echo "Fix /etc/issue..."
   cp -f /etc/nethserver-release /etc/issue
   echo -e 'Kernel \\r on an \\m\n' >> /etc/issue
+  echo 'To login to NethServer please use following URL(s):' >> /etc/issue
+  ifconfig | grep -Eo 'inet (addr:)?([0-9]*\.){3}[0-9]*' | grep -Eo '([0-9]*\.){3}[0-9]*' | grep -v '127.0.0.1' | sed 's/.*/https:\/\/&:980/' >> /etc/issue
```

Comment:
I'd use the `ip` command: it's output is more parsable than `ifconfig`.

Reply:
I have two alternatives for you, maybe they could be more optimized but they should work. What do you think?

ip addr | grep -oP '(?<=inet\s)\d+(\.\d+){3}' | grep -v 127.0.0.1 | sed 's/.*/https:\/\/&:980/'

hostname -I | sed -E -e 's/[[:blank:]]+/\n/g' | sed 's/.*/https:\/\/&:980/' | grep -v 'https://:980'

Reply:
I prefer the `ip addr` one! With `-o` (oneline) option it's easier to parse:

    ip -o -4 addr | tr -s ' /' $'\t' | cut -f 4 | grep -vF 127.0.0.1 | xargs -I IP -- echo https://IP:980